pyproject.toml

```diff
@@ -93,6 +93,7 @@ norecursedirs = [
 [tool.ruff]
 fix = true
 target-version = "py39"
+extend-exclude = ["**/migrations/*.py"]
```

Comment:
pre-commit annoyed me with complaining about missing type annotations in the migration. Since I didn't quite feel like manually touching an autogenerated file, I decided to just add it to the list of excludes.